src/applications/hca/config/chapters/militaryService/additionalInformation.js

```diff
@@ -0,0 +1,76 @@
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+import MilitaryPrefillMessage from 'platform/forms/save-in-progress/MilitaryPrefillMessage';
+
+const {
+  campLejeune,
+  disabledInLineOfDuty,
+  exposedToRadiation,
+  isFormerPow,
+  postNov111998Combat,
+  purpleHeartRecipient,
+  radiumTreatments,
+  swAsiaCombat,
+  vietnamService,
+} = fullSchemaHca.properties;
+
+export default {
+  path: 'military-service/additional-information',
+  title: 'Service history',
+  uiSchema: {
+    'ui:title': 'Service history',
+    'ui:description': MilitaryPrefillMessage,
+    'view:textObject': {
+      'ui:description': 'Check all that apply to you.',
+    },
+    purpleHeartRecipient: {
+      'ui:title': 'Purple Heart award recipient',
+    },
+    isFormerPow: {
+      'ui:title': 'Former Prisoner of War',
+    },
+    postNov111998Combat: {
+      'ui:title':
+        'Served in combat theater of operations after November 11, 1998',
+    },
+    disabledInLineOfDuty: {
+      'ui:title':
+        'Discharged or retired from the military for a disability incurred in the line of duty',
+    },
+    swAsiaCombat: {
+      'ui:title':
+        'Served in Southwest Asia during the Gulf War between August 2, 1990, and Nov 11, 1998',
+    },
+    vietnamService: {
+      'ui:title': 'Served in Vietnam between January 9, 1962, and May 7, 1975',
+    },
+    exposedToRadiation: {
+      'ui:title': 'Exposed to radiation while in the military',
+    },
+    radiumTreatments: {
+      'ui:title':
+        'Received nose/throat radium treatments while in the military',
+    },
+    campLejeune: {
+      'ui:title':
+        'Served on active duty at least 30 days at Camp Lejeune from January 1, 1953, through December 31, 1987',
+    },
+  },
```

Comment:
Does this page do anything? All I see are titles.

Reply:
Good question that code did look pretty odd and on GH I didn't see where the `vets-json-schema` was being pulled in, but it looks like just a bunch of boolean values... Could probably be refactored into a `checkbox group` but what are the pros & cons of a `checkbox group`? Would it add value?

![image](https://user-images.githubusercontent.com/23741323/106155356-94fec500-614e-11eb-8688-b87c5caac74a.png)

Reply:
When the checkbox group is used, it wraps all the checkboxes in a single `<fieldset>`, I think how it is now, each checkbox is wrapped in it's own `<fieldset>` which isn't ideal.

Reply:
Roger that will create another issue for this refactor